doc/enterprise/audit_log.rst

```diff
@@ -1,40 +1,253 @@
+
+-   Filter by group. You can specify a list of :ref:`event groups <audit-log-event-groups>` to be recorded. For example,
+    select ``auth`` and ``priv`` to see the events related to authorization and granted privileges:
+
+    ..  code-block:: yaml
+
+        audit_log:
+          filter: [ auth,priv ]
+
+-   Filter by group and event. You can specify a group and a certain event depending on the purpose.
+    In the configuration below, ``user_create``, ``data_operations``, ``ddl``, and ``custom`` are selected to see the events related to
+
+    *   user creation
+    *   space creation, altering, and dropping
+    *   data modification or selection from spaces
+    *   :ref:`custom events <audit-log-custom>` (any events added manually using the audit module API)
+
+    ..  literalinclude:: /code_snippets/snippets/config/instances.enabled/audit_log/config.yaml
+        :language: yaml
+        :start-at: filter:
+        :end-at: custom ]
+        :dedent:
+
+..  _audit-log-format:
+
+Set the format of audit log events
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the :ref:`audit_log.format <configuration_reference_audit_format>` option to choose the format of audit log events
+-- plain text, CSV, or JSON.
+
+JSON is used by default. It is more convenient to receive log events, analyze them, and integrate them with other systems if needed.
+The plain format can be efficiently compressed.
+The CSV format allows you to view audit log events in tabular form.
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/audit_log/config.yaml
+    :language: yaml
+    :start-at: format:
+    :end-at: json
+    :dedent:
+
+..  _audit-log-spaces:
+
+Specify the spaces to be logged
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since :doc:`3.0.0 </release/3.0.0>`, :ref:`audit_log.spaces <configuration_reference_audit_spaces>` is used to specify
+a list of space names for which data operation events should be logged.
+
+In the configuration below, only the events from the ``bands`` space are logged:
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/audit_log/config.yaml
+    :language: yaml
+    :start-at: spaces:
+    :end-at: bands ]
+    :dedent:
+
+..  _audit-log-extract:
+
+Specify the logging mode in DML events
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since :doc:`3.0.0 </release/3.0.0>`, it is possible to
+force the audit subsystem to log the primary key instead of a full tuple in DML operations.
+To do so, set the :ref:`audit_log.extract_key <configuration_reference_audit_extract_key>` option to ``true``.
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/audit_log/config.yaml
+    :language: yaml
+    :start-at: extract_key:
+    :end-at: true
+    :dedent:
+
+..  _audit-log-run-read:
+
+Reading audit logs
+------------------
+
+In the example, the logs are written to the ``audit_tarantool.log`` file.
+Let's create a space ``bands`` and check the logs in the file after the creation:
```

Comment:
> Let's create

Am I correct that we try to avoid first and second person writing?

https://www.tarantool.io/en/doc/latest/contributing/docs/style/#don-t-say-we

We can change to smth like this:

_In the example below, the `***` function creates the `***` ..._